apps/user/src/services/main/queries.ts

```diff
@@ -9,8 +9,11 @@ interface MainNoticeListType {
 }
 
 export const useMainNoticeListQuery = () => {
-    const { data } = useQuery<MainNoticeListType[]>([KEY.MAIN_NOTICE_LIST], () => mainNoticeList());
-    return { data: data || [] };
+    return useQuery<MainNoticeListType[]>([KEY.MAIN_NOTICE_LIST], () => mainNoticeList(), {
+        refetchOnWindowFocus: false,
+        initialData: [],
+        staleTime: 1000 * 60,
+    });
```

Comment:
queryClient에서 refetchOnWindowFocus와 staleTime를 설정하는 것도 반복을 제거하는 방법일 거 같아요.

```suggestion
    return useQuery<MainNoticeListType[]>([KEY.MAIN_NOTICE_LIST], () => mainNoticeList(), {
        initialData: [],
    });
```

Reply:
오 공통적인 부분을 줄일수있다니 너무 좋네요 !! 감사합니다 !

Reply:
혹시 queryClient는 전역으로 관리하는것이 좋을까요? 만약 전역으로 관리한다면 패키지로 모든 웹사이트를 전역으로 관리하는것도 괜찮을까요?